src/main/java/it/gov/pagopa/pu/migration/wf/activity/ingestion/organizations/OrganizationsMigrationFileTypeHandlerActivityImpl.java

```diff
@@ -63,11 +67,15 @@ protected MigrationFileResult handleRetrievedFiles(List<Path> retrievedFiles, Up
 
     List<IngestionFlowFile> filesUploaded = new ArrayList<>(retrievedFiles.size());
     for (Path file : retrievedFiles) {
+      Path zipFilePath = file.getParent().resolve(file.getFileName() + ".zip");
```

Comment:
file.getFileName() contains the extension, this way you are not replacing it with .zip, but you are concatenating the extensions